docs/products/metrics/howto/migrate-influxdb-thanos.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Migrate Aiven for InfluxDB to Aiven for Metrics
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Migrate your Aiven for InfluxDB databases to Aiven for Thanos Metrics using the Aiven Console migration tool.
+
+## Compatibility considerations
+
+Before migrating an InfluxDB database to Aiven for Metrics, carefully
+review your current InfluxDB setup.
+
+- **Review database setup:** Examine your InfluxDB database's data structures, storage
+  patterns, and configurations. Identify any unique features, custom settings, and
+  specific configurations.
+- **API and query language compatibility:** Thanos Metrics uses PromQL, which is
+  different from InfluxDB's InfluxQL. Update your queries to be
+  compatible with PromQL. For assistance, you can use tools such as the
+  [InfluxQL to PromQL converter](https://github.com/logzio/influxql-to-promql-converter)
+  or Aiven's own [dashboard converter](https://github.com/Aiven-Open/influxql-to-m3-dashboard-converter).
+  The built-in dashboards provided by Aiven services through integrations will not
+  require any manual changes.
+- **Version compatibility:** Ensure that your InfluxDB version supports migration to
+  Thanos Metrics. You might need to upgrade your InfluxDB to a compatible version.
+
+## Prerequisites
+
+Before starting the migration from an Aiven for InfluxDB service:
+
+- Make a note of the Aiven project and Aiven for InfluxDB service name
+  for migration in the [Aiven Console](https://console.aiven.io/).
+
+The [Aiven Console](https://console.aiven.io/)migration tool automatically uses
+connection details like the hostname, port, and credentials linked to the selected
+Aiven for InfluxDB service.
+
+## Database migration steps
+
+1. Log in to the [Aiven Console](https://console.aiven.io/) and select
+   the Aiven for Metrics service for your InfluxDB database migration.
+1. Go to **Service settings** from the sidebar.
+1. Scroll to the **Service management** section, and
+   click <ConsoleLabel name="actions"/> > **Import database**.
+   to initiate the import process.
+1. Follow the wizard to guide you through the database migration process.
+
+### Step 1: Configure
+
+1. From the drop-down menu, select your project name.
```

Comment:
I think we can get away with not describing the UI for this and the following step.
```suggestion
1. Select your project name.
```